<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddFileDate</class>
 <widget class="QDialog" name="DialogAddFileDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un fichier</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonBrowse">
     <property name="toolTip">
      <string>SÃ©lectionner un fichier</string>
     </property>
     <property name="text">
      <string>Choisir un ou plusieurs fichiers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetOneFile" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fichier</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFilePath"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDateYear">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetFileDate">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fichier</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>126</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAddFileDate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAddFileDate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
